en/08_Changelogs/5.3.0.md

```diff
@@ -82,9 +82,13 @@ Validation can be added to a content block using standard [Model Validation and
 
 Elemental data is no longer sent when saving the parent `DataObject` (usually a `Page`) that contains an [`ElementalAreaField`](api:DNADesign\Elemental\Forms\ElementalAreaField). Instead, when saving the parent `DataObject`, all the child inline-editable elements that have unsaved changes are inline saved at the same time. This change was done to consolidate the saving process down to a single code path. The code that was previously used to process any element data sent with the parent data has been removed.
 
-### Define scaffolded form fields for relations to `DataObject` models {#scaffolded-relation-formfields}
+### `FormField` scaffolding for `DataObject` models {#scaffolding}
 
-Most `DataObject` classes will rely on some amount of automatic scaffolding of form fields in their [`getCMSFields()`](api:SilverStripe\ORM\DataObject::getCMSFields()) implementations. However, it's common for modules to provide a specialised form field which is intended to always be used with a specific `DataObject` class. In those cases, even though you always want to use that form field with that class, you have to copy some boilerplate code from the module's documentation to set it up.
+We've made a few improvements to how form fields are scaffolded for [`DataObject::getCMSFields()`](api:SilverStripe\ORM\DataObject::getCMSFields()).
+
+#### Define scaffolded form fields for relations {#scaffolding-relation-formfields}
+
+Most `DataObject` classes will rely on some amount of automatic scaffolding of form fields in their `getCMSFields()` implementations. However, it's common for modules to provide a specialised form field which is intended to always be used with a specific `DataObject` class. In those cases, even though you always want to use that form field with that class, you have to copy some boilerplate code from the module's documentation to set it up.
```

Comment:
The only change to this line (see removal of the same line above) was removing the API link since we're linking to that in the parent heading section instead.